src/storage/cmemlog.cc

```diff
@@ -39,6 +39,17 @@ void memlog_add_table_mapping(int db_id, int root_page_id, char *table_name) {
   table_map[db_id][root_page_id] = std::string(table_name);
 }
 
+void memlog_add_column_change(int db_id, int root_page_id, char* column_info) {
+  std::string new_column = std::string(column_info); 
+  std::vector<std::string> result; 
+  std::istringstream iss(new_column); 
+  std::string s;
+  while (iss >> s) { 
+    result.push_back(s);
+    std::cout << "word: " << s << std::endl;
```

Comment:
Is this a debug message? if yes make that explicit through a comment or a debug flag.
Even better you could use something like glog to have debug and prod logs